documentation/api/addAssertion.md

```diff
@@ -232,9 +220,11 @@ expect.addAssertion('<array> [not] to have item <any>', function (
 In this way, when `to have item` is invoked with the `not` flag, that flag will
 be passed along to `to contain`.
 
-When flags are propagated, one can also invert the flag as follows:
+When flags are propagated, one can also invert the flag. This means that if
+`to have item` is invoked with the `not` flag, that flag will not be propagated
+to `to contain` - and vice versa:
```

Comment:
This is not really correct, it is the negation that is propagated to `to contain`.

Reply:
Argh, I just smashed two sentences from the original together - will correct this!